about.html

```diff
@@ -80,11 +151,12 @@ <h2>Partners</h2>
                 <li><img src="./img/daum.png" alt="partners copany"></li>
                 <li><img src="./img/airbnb.png" alt="partners copany"></li>
             </ul>
-            </div>
-    </section> 
+        </div>
+    </section>
     <footer>
```

Comment:
Please change the background color of your footer in your about to a dark background so that your footer meets the requirements 🤓